databases/week1/db1-homework-my_squirrel.sql

```diff
@@ -0,0 +1,85 @@
+-- Hello My_SQuirreLs, here is my homework ---> databases-1 (a.k.a. week 15) ---> Magda /HYF#12
+
+USE hyf_db_1;   -- all the comments/questions will be removed once resolved :) 
+SELECT * FROM status; 
+SELECT * FROM task;  --  jest powiązany z tabelą status przez status_id i z tabelą user przez user_id
+SELECT * FROM user;   
+
+-- 01 -- Find out how many tasks are in the task table: 
+SELECT count(id) AS `number_of_tasks_in_task_table`   -- backticks work with column aliases but not the values!!!
+FROM task;
+-- Question: Is the alias too long? Or should I leave it as descriptive as possible?
+-- Question: Would count(*) be too redundant? 
+
+-- 02 -- Find out how many tasks in the task table do not have a valid due date: 
+SELECT count(id) AS `tasks_in_task_table_without_valid_due_date`  
+FROM task 
+WHERE due_date IS NULL;  
+
+-- 03 -- Find all the tasks that are marked as done: 
+SELECT task.*, status.name
+FROM task INNER JOIN status 
+ON task.status_id = status.id 
+WHERE status.name = 'Done';  -- [LIKE] equals [=] 
+-- or --
+SELECT task.*, status.id
+FROM task INNER JOIN status 
+ON task.status_id = status.id 
+WHERE status.id = 3; 
+-- or -- 
+SELECT * 
+FROM task INNER JOIN status 
+ON task.status_id = status.id
+WHERE status.name = 'Done';
+
+-- 04 -- Find all the tasks that are not marked as done   
+SELECT task.*, status.name
+FROM task INNER JOIN status 
+ON task.status_id = status.id 
+WHERE status.name NOT LIKE 'Done';  -- [NOT LIKE] equals [!=] 
+-- or --
+SELECT task.*, status.id
+FROM task INNER JOIN status 
+ON task.status_id = status.id 
+WHERE status.id NOT LIKE 3; 
```

Comment:
LIKE works with numbers? :O 
You guys are always teaching me stuff :D 
In this case I would do ` WHERE status.id <> 3`

Reply:
`!= ` instead of `<>` also works btw

Reply:
Oh cool, thank you! 
I have been playing / experimenting and that seems to be the conclusion :) 
...so just to remember the options, I like to put alternative versions while I'm learning.